Source/WebCore/rendering/RenderReplaced.cpp

```diff
@@ -452,9 +452,11 @@ void RenderReplaced::computeIntrinsicSizesConstrainedByTransferredMinMaxSizes(Re
             };
 
             auto [minLogicalWidth, maxLogicalWidth] = computeMinMaxLogicalWidthFromAspectRatio();
+            maxLogicalWidth = std::max(minLogicalWidth, maxLogicalWidth);
```

Comment:
I'd expect a function named "computeMIN_MAX" return a pair of _min_ and _max_ values. Adjustments like this is highly error prone (e.g. if I added this function call somewhere, I would certainly not think about doing an extra std::max right after) -also why is it adjusting to the max direction?

Reply:
Moved the logic inside each of the methods.

The spec says that the minimum transferred value is computed first. Then, the maximum value is computed and floored by the previously computed minimum (if that is what you are asking)

Reply:
> Moved the logic inside each of the methods.
> 
> The spec says that the minimum transferred value is computed first. Then, the maximum value is computed and floored by the previously computed minimum (if that is what you are asking)
Sure, what I was trying to imply was that it's hard to reason about that line without knowing the context and people start copy-pasting the code without knowing what it does.